docs/non_containerized_NSFS.md

```diff
@@ -642,3 +642,60 @@ Access is restricted to these whitelist IPs. If there are no IPs mentioned all I
 ```
 sudo noobaa-cli whitelist --ips '["127.0.0.1", "192.000.10.000", "3002:0bd6:0000:0000:0000:ee00:0033:000"]'  2>/dev/null
 ```
+
+## Anonymous request
+
+Anonymous requests are those sent without access and secret key and Noobaa NSFS allow it with supporting bucket policy. Users have to create an anonymous account before accessing the resources. Noobaa uses anonymous account's uid and gid or user(distinguished_name) for accessing bucket storage paths. Noobaa can have one anonymous account. Commands for creating anonumous account are
+```
+node src/cmd/manage_nsfs account add --anonymous --uid {uid} --gid {gid}
```

Comment:
can someone create/update a regular account called anonymous? we need to block it if so